client/orm/models.go

```diff
@@ -389,12 +392,6 @@ func (mc *_modelCache) register(prefixOrSuffixStr string, prefixOrSuffix bool, m
 					}
 				}
 			}
-
-			if mi.fields.pk == nil {
```

Comment:
I think output some warn information would be better

Reply:
What kind of warning message is output?

Reply:
("<orm.RegisterModel> `%s`lack of a primary key field, default is to use 'id' if not set\n", name)

Reply:
ok